virt/release_notes/virt-4-15-release-notes.adoc

```diff
@@ -0,0 +1,266 @@
+** As a workaround, you can restart the `ceph-mgr` to purge the VM clones.
+
+//CNV-34198 (BZ 2237287)
+* If you use Portworx as your storage solution on AWS and create a VM disk image, the created image might be smaller than expected due to the filesystem overhead being accounted for twice. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2237287[*BZ#2237287*])
+** As a workaround, you can manually expand the Persistent Volume Claim (PVC) to increase the available space after the initial provisioning process completes.
+
+//BZ2216038 from Ying
+* If you simultaneously clone more than 1000 VMs using the provided DataSources in the `openshift-virtualization-os-images` namespace, it is possible that not all of the VMs will move to a running state. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2216038[*BZ#2216038*])
+** As a workaround, deploy VMs in smaller batches.
+
+//BZ2247593 from Ying
+* Live migration cannot be enabled for a virtual machine instance (VMI) after a hotplug volume has been added and removed. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2055595[*BZ#2247593*])
+
+[discrete]
+[id="virt-4-15-ki-virtualization"]
+==== Virtualization
+//4.14 Leave in per Stu
+* {VirtProductName} links a service account token in use by a pod to that specific pod. {VirtProductName} implements a service account volume by creating a disk image that contains a token. If you migrate a VM, then the service account volume becomes invalid. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2037611[*BZ#2037611*])
+** As a workaround, use user accounts rather than service accounts because user account tokens are not bound to a specific pod.
+
+// Leave in for 4.14 per Stu; remove from 4.15 RNs and move content to CNV docs, per Avital
+* With the release of the link:https://access.redhat.com/errata/RHSA-2023:3722[RHSA-2023:3722] advisory, the TLS `Extended Master Secret` (EMS) extension (link:https://datatracker.ietf.org/doc/html/rfc7627[RFC 7627]) is mandatory for TLS 1.2 connections on FIPS-enabled RHEL 9 systems. This is in accordance with FIPS-140-3 requirements. TLS 1.3 is not affected. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2157951[*BZ#2157951*])
++
+Legacy OpenSSL clients that do not support EMS or TLS 1.3 now cannot connect to FIPS servers running on RHEL 9. Similarly, RHEL 9 clients in FIPS mode cannot connect to servers that only support TLS 1.2 without EMS. This in practice means that these clients cannot connect to servers on RHEL 6, RHEL 7 and non-RHEL legacy operating systems. This is because the legacy 1.0.x versions of OpenSSL do not support EMS or TLS 1.3. For more information, see link:https://access.redhat.com/solutions/7018256[TLS Extension "Extended Master Secret" enforced with Red Hat Enterprise Linux 9.2].
++
+As a workaround, upgrade legacy OpenSSL clients to a version that supports TLS 1.3 and configure {VirtProductName} to use TLS 1.3, with the `Modern` TLS security profile type, for FIPS mode.
```

Comment:
Did we decide how to format workarounds going forward? Some of them have the sub-bullet and others are like this. I remember some vague discussion about it but I totally forget where it landed.

Reply:
I think historically we have formatted workarounds as sub-bullets. I don't remember if we had decided to format them in a different way going forward. Personally, I am fine with the current formatting because it helps the workarounds stand out visually.